requirements/base.txt

```diff
@@ -4,7 +4,7 @@
 #
 #    make upgrade
 #
-asgiref==3.4.1
+asgiref==3.5.0
```

Comment:
Dropped support for `Python 3.6`.  No further major change: https://github.com/django/asgiref/blob/main/CHANGELOG.txt